README.md

```diff
@@ -1,15 +1,31 @@
-# ipfs.io/refs
+# refs
 
-Tool for building and publishing the DAG at https://ipfs.io/refs
+> DMCA notices, and tools for publishing them
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
-[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/) [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+Tool for building and publishing the DAG at https://ipfs.io/refs
 
 Renders a DAG from various data sources:
 
 - DMCA notices served for the gateway at ipfs.io -- https://github.com/ipfs/refs-denylists-dmca
 - TODO: Content archived on our storage hosts -- https://github.com/ipfs/refs-solarnet-storage
 
-# Usage
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [License](#license)
+
+## Install
+
+Clone this repo. Depends on [npm](https://npmjs.com) and [node.js](https://nodejs.com).
```

Comment:
Nope this doesn't depend on npm and node.js -- I guess this is because there's a package.json here used by gx.